language/move-compiler/src/to_bytecode/optimize/remove_fallthrough_jumps.rs

```diff
@@ -2,20 +2,30 @@
 // Copyright (c) The Move Contributors
 // SPDX-License-Identifier: Apache-2.0
 
-use move_ir_types::ast as IR;
+use crate::parser::ast::FunctionName;
+use move_ir_types::ast::{self as IR};
 use std::collections::{BTreeSet, HashMap};
 
 // Removes any "fall through jumps", i.e. this a is a jump directly to the next instruction.
 // Iterates to find a fixpoint as it might create empty blocks which could create more jumps to
 // clean up
 
-pub fn code(loop_heads: &BTreeSet<IR::BlockLabel_>, blocks: &mut IR::BytecodeBlocks) {
-    let mut changed = true;
-    while changed {
+pub fn optimize(
+    _f: &FunctionName,
+    loop_heads: &BTreeSet<IR::BlockLabel_>,
+    _locals: &mut Vec<(IR::Var, IR::Type)>,
+    blocks: &mut IR::BytecodeBlocks,
+) -> bool {
+    let mut changed = false;
+    loop {
```

Comment:
This fixpoint should be redundant given there's an outer one. @tnowacki, thoughts on removing it and relying on the outer optimization alone?